### DISCUSSION TITLE
blog/change-vs-code-sidebar-font/

### DISCUSSION THREAD

**Discussion by giscus** [armno/armno.in.th#52] [Blog Post Comments]
# blog/change-vs-code-sidebar-font/

Changing the sidebar font in VSCode can be done only via extensions. I'm using APC extension to achieve that.

https://armno.in.th/blog/change-vs-code-sidebar-font/

**Comment by lazycatx64** [comment]
APC dont work anymore with VSCode version above 1.93 since last year
https://github.com/drcika/apc-extension/issues/225

**Reply by armno** [reply]
Hey @lazycatx64. Thanks for the heads up.  I have added a note in this post that this is no longer working.

**Comment by g-e-n-v** [comment]
Another alternative to APC: https://marketplace.visualstudio.com/items?itemName=subframe7536.custom-ui-style

```json
{
    "custom-ui-style.font.sansSerif": "iA Writer Mono V",
	"custom-ui-style.font.monospace": "'GeistMono Nerd Font', monospace"
}
```
